Documentation/EyeTracking/EyeTracking_Main.md

```diff
@@ -1,10 +1,16 @@
 ![Eye Tracking in MRTK](../../Documentation/Images/EyeTracking/mrtk_et_compilation.png)
 # Eye Tracking in the Mixed Reality Toolkit
 
-The [Mixed Reality Toolkit](https://github.com/Microsoft/MixedRealityToolkit-Unity) (MRTK) supports amongst others _'HoloLens 2'_ which offers an exciting and powerful new input: Eye Tracking! 
-Eye Tracking enables users to quickly and effortlessly engage with holograms across their view and can make your system smarter by better identifying a user's intention.
-New to Eye Tracking? No problem! We have created a number of videos, tutorials and samples to get you started! 
-1. [Getting started with Eye Tracking in MRTK](EyeTracking_BasicSetup.md)
+_HoloLens 2_ offers an exciting and powerful new input: Eye tracking!
+Eye tracking enables users to quickly and effortlessly engage with holograms across their view and can make your system smarter by better identifying a user's intention. Check out Microsoft's Mixed Reality [documentation on eye tracking on HoloLens 2](https://docs.microsoft.com/en-us/windows/mixed-reality/eye-tracking) for more details, such as explaining powerful applications and design guidelines for eye tracking in mixed reality. 
+
+New to Eye Tracking? No problem! We have created a number of videos, tutorials and samples to get you started in the [Mixed Reality Toolkit](https://github.com/Microsoft/MixedRealityToolkit-Unity) (MRTK)! 
```

Comment:
```suggestion
New to eye tracking? No problem! We have created a number of videos, tutorials and samples to get you started in the [Mixed Reality Toolkit](https://github.com/Microsoft/MixedRealityToolkit-Unity) (MRTK)! 
```

Reply:
There are a few more instances - you might want to do a 

find . -type f -exec sed -i 's/Eye Tracking/eye tracking/g' {} +

(If you have git bash). If you're not using git bash then there's something else probably.

Obviously do this only with a fresh workspace (I don't know what other places actually don't need this treatment applied)

Reply:
Yeah, it's a bit tricky, because sometimes it's at the beginning of a sentence and the Eye is uppercase. So, don't want to just blindly replace. Hope I got them all now though.